extension/main.js

```diff
@@ -16,13 +16,31 @@
 	}
 
 	function update() {
-		gitHubNotifCount(function (count) {
+		gitHubNotifCount(function (count, interval) {
+			var period = 1;
+			if (interval !== GitHubNotify.settings.get('interval')) {
+				GitHubNotify.settings.set('interval', interval);
+				period = Math.ceil(interval / 60);
+				if (period < 1) {
```

Comment:
Actually, scratch the above comment as well.

The root of the problem is that `GitHubNotify.settings.get('interval')` is a string, so it will never `=== interval`. Either convert that to a number, or make sure the value you get from settings is always a number.